lisp/mastodon-tl.el

```diff
@@ -54,7 +54,10 @@
 (defun mastodon-tl--get-tag-timeline ()
   "Prompts for tag and opens its timeline."
   (interactive)
-  (let ((tag (read-string "Tag: ")))
+  (let* ((word (word-at-point))
```

Comment:
Currently, if you're `(word-at-point)` is nil, then the prompt looks like `Tag(nil): `.

It can be fixed with a small tweak:

```elisp
(word (or (word-at-point) ""))
```

This will look like `Tag(): `.

I'd like to optionally display the parens, but I think it can be fixed in later commits.